harness/determined/_swagger/client/api_client.py

```diff
@@ -14,18 +14,18 @@
 import datetime
 import json
 import mimetypes
-from multiprocessing.pool import ThreadPool
```

Comment:
Please discard changes from `harness/determined/_swagger/` and rebase onto tip-of-master; there was a bug in `make fmt` that caused this